tidepods/generate_pts.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import rasterio
+import rasterio.warp
+from shapely.geometry import box, Point
+
+
+def transform_shape(infile):
+    """Reads AOI bounds and reprojects to EPSG:4326. Returns bounds as shapely polygon
+
+    Parameters
+    ----------
+    infile : str
+        File path to raster AOI.
+
+    Returns
+    -------
+    shp : shapely object
+
+    """
+    with rasterio.open(infile) as src:
```

Comment:
For the record: You don't need `rasterio` for this (which is as heavyweight as Python packages get). You could achieve the same via `PyProj` or even `fiona`. But I think it's ok to keep that for now.

Reply:
Kept for now. Moved the rasterio import to within the function so it is only imported in cases where the input file is a raster. Else fiona is imported if the input file is a vector.